src/pretix/base/services/checkin.py

```diff
@@ -155,3 +157,16 @@ def perform_checkin(op: OrderPosition, clist: CheckinList, given_answers: dict,
             'datetime': dt,
             'list': clist.pk
         }, user=user, auth=auth)
+
+
+@receiver(order_placed, dispatch_uid="autocheckin_order_placed")
+def order_placed(sender, **kwargs):
+    order = kwargs['order']
+    event = sender
+
+    cls = list(event.checkin_lists.filter(auto_checkin_sales_channels__contains=order.sales_channel).prefetch_related(
+        'limit_products'))
+    for op in order.positions.all():
```

Comment:
One small performance improvement in case this feature is not used at all:
```suggestion
    if not cls:
        return
    for op in order.positions.all():
```